tests/fixture/environment.go

```diff
@@ -20,16 +22,26 @@ func NewTestEnvironment(
 ) *e2e.TestEnvironment {
 	// Run only once in the first ginkgo process
 	nodes := tmpnet.NewNodesOrPanic(flagVars.NodeCount())
+	nodes[0].Flags["http-port"] = "9650"
+
 	subnet := NewHyperVMSubnet(
 		vmName,
 		vmID,
 		genesisBytes,
 		nodes...,
 	)
 	network := NewTmpnetNetwork(owner, nodes, subnet)
-	return e2e.NewTestEnvironment(
+
+	testEnv := e2e.NewTestEnvironment(
 		testContext,
 		flagVars,
 		network,
 	)
+
+	chainID := network.GetSubnet(vmName).Chains[0].ChainID
+	if chainID != ids.Empty {
```

Comment:
On the first run, the chainID will be populated, but on subsequent runs, it will be empty, leading us to attempt creating an alias for an empty chainID.

Reply:
Doesn't that mean that this fails to produce a stable identity on subsequent runs? If we use the AvalancheGo Info API to get the chainID instead, then I think we can reliably grab the actual chainID here

Reply:
@aaronbuchwald I'm running into some issues redoing this with `info` API. The `info.getBlockchainID` method requires a known alias, but to set an alias, we need a chainID. It's a bit of a chicken-and-egg situation. I also couldn't find anything helpful in the admin API.

Reply:
Ah you're right, we can use the P-Chain API though: https://docs.avax.network/api-reference/p-chain/api#platformgetblockchains

Reply:
Thanks! Deprecated, but works well for now